README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# THE NEWSWEEK
+
+This project is a clone of the Newsweek website at (newsweek.com). As a requirement of the program, basic HTML and CSS were used during the design. 
+
+
+![screenshot](./newsweek-screenshot.png)
+
+
+## Built With
+
+- HTML
+- CSS
+
```

Comment:
Add bootstrap in your readme, you will want the recruiters to know that you can use this tech